### DISCUSSION TITLE
My webpage is showing blank

### DISCUSSION THREAD

**Discussion by PATIENCE-HUB** [community/community#67234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Please my web page is showing blank. Kindly assist, as the link seen below

https://patience-hub.github.io/healthwise/

**Comment by radar-blip** [comment]
You need an index.html
You could rename one of the existing files to index.html
or just create a new index.html

**Comment by ALNPRA123** [comment]
you need index.html bro,if you create about a website page u can't forget about index.html because all syntax in your program can be run about index.html,just kidding or u don't know about this lol ,hope my answer can be you helpfully
